src/pudl/transform/eia.py

```diff
@@ -1149,133 +1140,171 @@ def fix_balancing_authority_codes_with_state(
     return plants
 
 
-@multi_asset(
+@asset(
     ins={
-        asset_id: AssetIn()
-        for asset_id in [
-            "clean_boiler_fuel_eia923",
+        table_name: AssetIn()
+        for table_name in [
             "clean_boiler_generator_assn_eia860",
-            "clean_boilers_eia860",
-            "clean_coalmine_eia923",
-            "clean_fuel_receipts_costs_eia923",
             "clean_generation_eia923",
-            "clean_generation_fuel_eia923",
-            "clean_generation_fuel_nuclear_eia923",
             "clean_generators_eia860",
-            "clean_ownership_eia860",
-            "clean_plants_eia860",
-            "clean_utilities_eia860",
+            "clean_boiler_fuel_eia923",
```

Comment:
I looked at the contents of the BGA function and removed all of the tables here that it wasn't accessing, which left only:
* `clean_boiler_generator_assn_eia860`
* `clean_generation_eia923`
* `clean_generators_eia923`
* `clean_boiler_fuel_eia923`

Does that sound right @cmgosnell?

Is there any additional information that would be useful in here from the much-expanded `boilers_eia860` table? (not that we should do it now, but maybe think about it a little and make an issue if so)